demo-shell/src/app/components/app-layout/cloud/tasks-cloud-demo.component.html

```diff
@@ -13,6 +13,9 @@
             [processInstanceId]="editedFilter.processInstanceId"
             [name]="editedFilter.taskName"
             [parentTaskId]="editedFilter.parentTaskId"
+            [owner]="editedFilter.owner"
+            [lastModifiedFrom]="editedFilter.lastModifiedFrom"
+            [lastModifiedTo]="editedFilter.lastModifiedTo"
```

Comment:
do we need 'priority' and 'standalone' added as well, here?

Reply:
priority yes 
standalone not, because is not working on activiti 7 side